src/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card__product-review {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews-count {
+  color: #060b35;
+  font-weight: 400;
```

Comment:
font-weight is inherited property
So, you can write font-weight: 400; for .card block 1 time and remove it from 3 selectors

Reply:
font-weight is not inherited property by default

Reply:
i was wrong :)
font-weight isn't inherited property)
my bad)